Main.py

```diff
@@ -10,53 +10,57 @@
 import sys
 import codecs
 
+EXIT = 99
+BACK = 66
+
 
 def NewsSources():
-    NewsSources=ConfigurationReader().GetWebsiteSupported()
+    NewsSources = ConfigurationReader().GetWebsiteSupported()
     return NewsSources
 
+
 def App():
-    newsSources=NewsSources()
+    newsSources = NewsSources()
     while True:
-        for i in xrange(len(newsSources)):
-            print ("["+str(i)+"]" +"\t" +newsSources[i])
-        print ("Please enter the index of the news source or press 99 to quit")
+        for i, newsSource in enumerate(newsSources):
+            print "[%s] \t %s " % (i, newsSource)
+        print "Please enter the index of the news source or press 99 to quit"
         try:
-            newsSourceNumber=raw_input("News Source Number >>>> ")
-        except ValueError:  
-            print ("That is not a valid News Source Number")
-        newsSourceNumber=int(newsSourceNumber)
-        if newsSourceNumber==99:
+            newsSourceNumber = raw_input("News Source Number >>>> ")
+        except ValueError:
+            print "That is not a valid News Source Number"
+        newsSourceNumber = int(newsSourceNumber)
+        if newsSourceNumber == EXIT:
             sys.exit()
-        if (newsSourceNumber >=len(newsSources)):
-            print ("Please select the index no less than "+ str(len(newsSources)))  

```

Comment:
removing parenthesis from the print doesn't make it compliant only for the python 2.7  while having it makes it work for both python 2.7 and python 3.

Reply:
If the aim is to make it compatible between python2 and python3, generally either
1) 'from __future__ import print_function' is used 
Or
2) six is made a part of the library. Refer http://six.readthedocs.io/ .

Reply:
I think it is is better to go with 'from future import print_function' at this point in time.